pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2243,6 +2245,15 @@ cdef class BYearEnd(YearOffset):
     """
     DateOffset increments between the last business day of the year.
 
+    Parameters
+    ----------
+    n : int, default 1
+        The number of years represented.
+    normalize : bool, default False
+        Normalize start/end dates to midnight before generating date range.
+    month : int, default 1
```

Comment:
ah missed this, the default is actually `12`, not `1` - fancy opening a follow-up to fix this?